pkg/bbgo/interact.go

```diff
@@ -61,10 +43,10 @@ func NewCoreInteraction(environment *Environment, trader *Trader) *CoreInteracti
 	}
 }
 
-func (it *CoreInteraction) FilterStrategyByInterface(checkInterface interface{}) (strategies []string, found bool) {
+func FilterStrategyByInterface(checkInterface interface{}, exchangeStrategies map[string]SingleExchangeStrategy) (strategies []string, found bool) {
```

Comment:
can you return map[string]SingleExchangeStrategy ? so that we can keep the object in the map.

Reply:
you can add another function called `strategySignatures(map[string]SingleExchangeStrategy) []string` to get signatures from the map.

Reply:
So we use strategy map obj to pass in/out functions for convenience in situations like we want to access the strategy objects. and with strategySignatures() we can easily get the string list of strategies
am i right?

Reply:
yes~

Reply:
done